tests/utils/common/constants/cluster.go

```diff
@@ -57,4 +57,14 @@ var (
 // instance type
 const (
 	DefaultInstanceType = "m5.xlarge"
+
+	M5XLarge  = "m5.xlarge"
+	M52XLarge = "m5.2xlarge"
+	M6gXLarge = "m6g.xlarge"
+)
+
+// cpu architecture
+const (
+	AMD = "amd"
```

Comment:
We should use `amd64` and `arm64` instead

Reply:
OK